src/data/css-quiz.ts

```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
     Explanation:
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+  {
+    Question:
+      "Which of the below CSS property is used to add a stroke in the text ?",
+    Answer: ":text-stroke",
+    Distractor1: ":text-transform",
+    Distractor2: ":text-decoration",
+    Distractor3: ":None of the above",
+    Explanation:
+      "The text-stroke is a CSS property that is used to add a stroke in the text. It is also supported by using WebKit.",
```

Comment:
I don't think we need that last sentence.
```suggestion
      "text-stroke is a CSS property that is used to add a stroke in the text.",
```